### ISSUE TITLE
agesa errors on apu1 when booting 4.11.0.5

### ISSUE THREAD

**Issue by 310b8fb5** [pcengines/coreboot#397]
After upgrading to 4.11.0.5 from 4.11.0.4 on an apu1 (including a power cycle), some errors are present in the console.

![screenshot](https://user-images.githubusercontent.com/61675653/79630535-a63b8f80-8141-11ea-8db0-4bda8e368325.png)

**Comment by miczyg1** [comment]
@310b8fb5 thank you for the report. We will investigate it.

**Comment by maxammann** [comment]
I can confirm this:

```
PC Engines apu1
coreboot build 20202604
BIOS version v4.11.0.6
ASSERTION ERROR: file 'src/vendorcode/amd/agesa/f14/Proc/Common/CommonReturns.c', line 174
ASSERTION ERROR: file 'src/vendorcode/amd/agesa/f14/Proc/CPU/cpuGeneralServices.c', line 817
ASSERTION ERROR: file 'src/vendorcode/amd/agesa/f14/Proc/Common/CommonReturns.c', line 174
ASSERTION ERROR: file 'src/vendorcode/amd/agesa/f14/Proc/CPU/cpuGeneralServices.c', line 817

```

**Comment by miczyg1** [comment]
This should be fixed in v4.12.0.1. Any udpates?

**Comment by 310b8fb5** [comment]
@miczyg1 this seems to be fixed. Thanks!
